framework/src/st_things/things_stack/framework/things_security_manager.c

```diff
@@ -60,9 +60,9 @@ typedef enum {
 	OIC_SEC_INVALID_PARAM = 2
 } OICSecurityResult;
 
-#if __MIPS__
+#if defined(__MIPS__)
 #define STRING_SVR_DB_PATH "/data1/oic_svr_db_server.dat"
-#elif __TIZEN__
+#elif defined(__TIZEN__)
 #define STRING_SVR_DB_PATH "/opt/data/ocfData/oic_svr_db_server.dat"
 #else
 #define STRING_SVR_DB_PATH "./oic_svr_db_server.dat"
```

Comment:
please update default DB PATH in TizenRT.

Reply:
@JeonginKim  Please can you explain this point?